docs/source/user-guide/key_concepts.rst

```diff
@@ -9,7 +9,7 @@ Check
 
 Each check enables you to inspect a specific aspect of your data and models.
 They are the basic building block of the deepchecks package, covering all kinds of common issues,
-such as: PerformanceOverfit, DataSampleLeakage, SingleFeatureContribution, DataDuplicates, and :ref:`many more checks <examples/checks>`.
+such as: PerformanceOverfit, DataSampleLeakage, SingleFeatureContribution, DataDuplicates, and :ref:`many more checks <examples-check-demonstrations>`.

```

Comment:
why is this the link?
isn't it still in examples/checks...?

Reply:
It's a label

Reply:
or actually i guess it should be to the :doc: directive
(or to a direct label that you added? if so then there is a general convention for them, but I guess doc directive is the best)